src/sql/base/browser/ui/panel/media/panel.css

```diff
@@ -149,7 +147,50 @@ panel {
 	font-weight: bold;
 	margin: 15px 5px 0px 5px;
 	line-height: 35px;
+	height: 35px;
 	border-style: solid;
 	border-width: 0 0 1px 0;
 	border-color: rgb(214, 214, 214);
 }
+
+.tabbedPanel .tab-group-heade {
+	display: block;
+}
+
+.tabbedPanel .tabLabelHidden .tab-group-header {
+	display: none;
+}
+
+.tabbedPanel .action-container {
+	display: flex;
+	flex-flow: row-reverse;
+}
+
+.tabbedPanel .tab-action {
+	width: 8px;
+	height: 8px;
+	padding: 3px;
+	cursor: pointer;
+}
+
+.tabLabelHidden .tabLabel {
+	display: none;
+}
+
+.vs .tabbedPanel .tab-action.collapse{
```

Comment:
nit: Format this file - there should be a space here

Reply:
this is by intentional, both class must be present for this rule to take effect. https://stackoverflow.com/questions/10036156/whats-the-difference-between-css-classes-foo-bar-without-space-and-foo-bar

Reply:
@alanrenmsft I was talking about the lack of a space after .tab-action.collapse 😄